src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,8 @@
-const numberToReversedDigits = (number) => {};
+function numberToReversedDigits(number) {
+    
+    const array = number.split("");
+    const numArray = array.map((element) => Number(element))
+    return numArray.reverse()
+}
```

Comment:
You can tidy this up a bit and avoid creating all the extra variables. Using the exact same logic you are already using, you can just chain the different methods together and return that result directly:
```
function numberToReversedDigits(number) {
  return number
    .split("")
    .map((element) => Number(element))
    .reverse();
}
```